cves/CVE-2015-5963.yml

```diff
@@ -66,46 +69,34 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: 575f59f9bc7c59a5e41a081d1f5f55fc859c5012
-  note:
+  note: |
+    (2015-08-05) In 1.4.x, ensure that cookies are not set in response when session
+    is empty. Also, prevent empty sessions from being created on flush(). Achieves
+    this by setting self._session_key to None. Added tests.
 - commit: 2f5485346ee6f84b4e52068c04e043092daf55f7
-  note:
+  note: |
+    (2015-08-05) In 1.7.x, ensure that cookies are not set in response when session
+    is empty. Also, prevent empty sessions from being created on flush(). Achieves
+    this by setting self._session_key to None. Added tests.
 - commit: 2eb86b01d7b59be06076f6179a454d0fd0afaff6
-  note:
+  note: |
+    (2015-08-05) In 1.8.x, ensure that cookies are not set in response when session
+    is empty. Added tests.
 vccs:
-- commit: 31ec68c5d939ee63852709163750a0516aaa2619
-  note: This VCC was discovered automatically via archeogit.
-- commit: 0d48087a53c84eb109f6c85472530899dfc94fc8
-  note: This VCC was discovered automatically via archeogit.
 - commit: 5e8efa9a6032f9c4278199ab354c3ff742387263
-  note: This VCC was discovered automatically via archeogit.
-- commit: '07889c13a63eeb3e8a73f1e02a21227def3ae548'
-  note: This VCC was discovered automatically via archeogit.
-- commit: c651b08f3981cb9c1e4631563675730a3577e0a9
-  note: This VCC was discovered automatically via archeogit.
+  note: |
+    (2008-08-13) flush() on session cleans out session and recreates key. Provide
```

Comment:
This feels out of place here, especially with no immediate context

Reply:
The important part of this commit is the recreation of a session on logout, which is the root cause of the issue where spamming logout will create too many sessions.